gradlew.bat

```diff
@@ -75,7 +75,9 @@ if "%ERRORLEVEL%"=="0" goto mainEnd
 :fail
 rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
 rem the _cmd.exe /c_ return code!
+pause
```

Comment:
Why was gradlew.bat changed? Is this intentional?

Reply:
> Added a few pauses to gradlew.bat so failures can be debugged easier.

Reply:
I ran into a few build fails and putting a few pauses into the fail section made it easier to debug. Probably don't need two pauses but I wasn't sure where the exit was.

Reply:
I would suggest leaving this kind of debug code in private branches

Reply:
That's completely fair.